app/views/user_mailer/delete_work_notification.html.erb

```diff
@@ -1,9 +1,12 @@
 <% content_for :message do %>
-  <p>Dear <%= @user.default_pseud.byline %>,</p>
-
-  <p>Your work <i><%= style_bold(@work.title.html_safe) %></i> was deleted at <%= (@work.pseuds.count > 1 && @user != User.current_user) ? (ts("the request of ") + style_pseud_link(User.current_user.default_pseud)).html_safe : "your request" %>.</p>
-  
-  <p>If you have questions, please <%= support_link("contact Support") %>.</p>  
-  
-  <p>Attached is a copy of your work for your reference.</p>
+<p><%= t('.hello', user:  @user.default_pseud.byline) %></p>
+<p><%= (@work.pseuds.count > 1 && @user != User.current_user) ?
+t('.part1_other',
+        title: "<i>#{style_bold(@work.title.html_safe)}</i>".html_safe,
+        pseud: style_pseud_link(User.current_user.default_pseud)).html_safe :
+t('.part1_yourself',
+        title: "<i>#{style_bold(@work.title.html_safe)}</i>".html_safe,
+        pseud: style_pseud_link(User.current_user.default_pseud)).html_safe %></p>
+<p><%= t('.part2', support: support_link(t 'user_mailer.delete_work_notification.support') ).html_safe %></p>
```

Comment:
There's an extra space before the final `)` on this line. Also, does the full `user_mailer.delete_work_notification.support` key need to be written out, or can it just be `.support`?